jx_sqlite/query_table.py

```diff
@@ -278,7 +278,7 @@ def query(self, query):
 
             output = Data(
                 meta={"format": "table"},
-                header=column_names,
+                header=[split_field(h)[0] if "\\" else h for h in column_names],
```

Comment:
We had a case (test_groupby_object_star) where `column_names = ['g\\.a', 'g\\.v', 'count']`
which results in Error: sets do not match. So I tried to change it to `['g.a', 'g.v', 'count']` using split_field(h). As it return field, as a list of dot separated fields, hence that 0th index(it will be only one).